components/Trade.js

```diff
@@ -41,7 +42,7 @@ const Trade = props => {
                   id: value.id,
                   name: value.filename,
                   preview: value.preview,
-                  selected: selectedItem === value.id,
+                  selected: selectedItem == value.id,
```

Comment:
Why single equals? If it's the wrong type it should be parsed before being set on the state.